.gitignore

```diff
@@ -30,3 +30,4 @@ sync-local.sh
 
 Userland/Libraries/LibWasm/Tests/Fixtures/SpecTests
 Userland/Libraries/LibWasm/Tests/Spec
+Base/home/anon/Games/
```

Comment:
No longer needed?

Reply:
Looks like the game requires assets from the original Half Life, so maybe let's keep this (but restrict it to `halflife`) and point out that requirement somewhere explicitly.

Reply:
Good point, I'll tack on the `halfilife`. Whereabouts shall I mention this requirement?